api/specs/web-server/_projects_states.py

```diff
@@ -34,35 +34,35 @@
 )
 
 
-def to_desc(exceptions: set[type[Exception]] | type[Exception]):
-    exc_classes = {exceptions} if not isinstance(exceptions, set) else exceptions
+def to_desc(exceptions: list[type[Exception]] | type[Exception]):
```

Comment:
aha, this was the culprit for the constant changes!  :-)
MINOR: i think it is simpler with positional arguments 
```python
def to_desc( *exceptions: type[Exception]):
    ...


# usage
to_desc(ProjectNotFoundError)
to_desc(ProjectNotFoundError, UserDefaultWalletNotFoundError)
to_desc(*exceptions)
```